raisinbread/instruments/eeg_acquisition_form.rules

```diff
@@ -0,0 +1,34 @@
+general_comments{-}Not Required{-}general_comments{@}=={@}NEVER_REQUIRED
+research_staff_two{-}Not Required{-}research_staff_two{@}=={@}NEVER_REQUIRED
+research_staff_three{-}Not Required{-}research_staff_three{@}=={@}NEVER_REQUIRED
+session_testing_room{-}Not Required{-}session_testing_room{@}=={@}NEVER_REQUIRED
+behavioural_assistant{-}Required if there is a behavioural assistant{-}behavioural_assistant_present{@}=={@}yes
+subject_medication_dosage{-}Required if there is an ongoing medication{-}subject_medication{@}=={@}yes
+head_measurement_unit{-}Required if head measurement was taken{-}head_measurement_taken{@}=={@}yes
+head_measurement{-}Required if head measurement was taken{-}head_measurement_taken{@}=={@}yes
+subject_away_60cm_from_monitor_reason{-}Required if subject cannot be 60cm away from montior{-}subject_away_60cm_from_monitor{@}=={@}no
```

Comment:
```suggestion
subject_away_60cm_from_monitor_reason{-}Required if subject cannot be 60cm away from monitor{-}subject_away_60cm_from_monitor{@}=={@}no
```

Reply:
@regisoc just spotted another typo to correct - thanks  (message appears in front end)